src/zenml/utils/analytics_utils.py

```diff
@@ -136,6 +136,17 @@ class AnalyticsEvent(str, Enum):
     ZENML_SERVER_DEPLOYED = "ZenML server deployed"
     ZENML_SERVER_DESTROYED = "ZenML server destroyed"
 
+    # ZenML Hub events
+    ZENML_HUB_PLUGIN_LIST = "ZenML Hub plugins listed"
+    ZENML_HUB_LOGIN = "ZenML Hub login"
+    ZENML_HUB_LOGIN_GH = "ZenML Hub login via GitHub"
```

Comment:
Along the same lines: Login/Logout? If anything, these should be tracked serverside. But why would I even care if someone logs out, I really don't get it.